docs/source/nodeoperations/restore.md

```diff
@@ -2,88 +2,223 @@
-```bash
-tar -ztvf task_287791d9-c257-4850-aef5-7537d6e69d90_tag_sm_20201228145917UTC_schema.tar.gz
--rw------- 0/0           12671 2020-12-28 13:17 users.cql
--rw------- 0/0            2216 2020-12-28 13:17 system_auth.cql
--rw------- 0/0             921 2020-12-28 13:17 system_distributed.cql
--rw------- 0/0           12567 2020-12-28 13:17 system_schema.cql
--rw------- 0/0            4113 2020-12-28 13:17 system_traces.cql
+To check progress use:
+```console
+$ sctool progress -c scylla/target restore/57228c52-7cf6-4271-8c8d-d446ff160747
+Restore progress
+Run:            0dd20cdf-abc4-11ee-951c-6e7993cf42ed
+Status:         DONE - restart required (see restore docs)
+Start time:     05 Jan 24 12:15:02 UTC
+End time:       05 Jan 24 12:15:09 UTC
+Duration:       6s
+Progress:       100% | 100%
+Snapshot Tag:   sm_20240105115931UTC
+
++---------------+-------------+----------+----------+------------+--------+
+| Keyspace      |    Progress |     Size |  Success | Downloaded | Failed |
++---------------+-------------+----------+----------+------------+--------+
+| system_schema | 100% | 100% | 214.150k | 214.150k |   214.150k |      0 |
++---------------+-------------+----------+----------+------------+--------+
 ```
 
-Extract this archive and copy each schema file to one of the cluster Pods by:
-```bash
-kubectl -n scylla cp users.cql simple-cluster-us-east-1-us-east-1a-0:/tmp/users.cql -c scylla
+As suggested in the progress output, we need to execute a rolling restart of ScyllaCluster.
+```console
+kubectl patch scyllacluster target --type merge -p '{"spec": {"forceRedeploymentReason": "schema restored"}}'
 ```
 
-To import schema simply execute:
-```bash
-kubectl -n scylla exec simple-cluster-us-east-1-us-east-1a-0 -c scylla -- cqlsh -f /tmp/users.cql
-```
+And wait until it's finished:
+```console
+$ kubectl wait --for='condition=Progressing=False' -n scylla scyllaclusters.scylla.scylladb.com/target
+scyllacluster.scylla.scylladb.com/target condition met
 
-Once the schema is recreated we can proceed to downloading data files.
+$ kubectl wait --for='condition=Degraded=False' -n scylla scyllaclusters.scylla.scylladb.com/target
+scyllacluster.scylla.scylladb.com/target condition met
 
-First let's save a list of snapshot files to file called `backup_files.out`:
+$ kubectl wait --for='condition=Available=True' -n scylla scyllaclusters.scylla.scylladb.com/target
+scyllacluster.scylla.scylladb.com/target condition met
+```
 
-```bash
-kubectl -n scylla-manager exec deployment.apps/scylla-manager-controller -- sctool backup files -c simple-cluster -L s3:backups -T sm_20201228145917UTC > backup_files.out
+Once completed, we can proceed to restoring the data.
+
+To restore tables content, create a restore task manually on target ScyllaCluster by executing following command in Scylla Manager Pod.
+```console
+sctool restore -c <CLUSTER_ID> -L <BACKUP_LOCATION> -T <SNAPSHOT_TAG> --restore-tables
```

Comment:
ditto `kubectl exec`

Reply:
you're using variables in some of the commands and set values in the others, let's make this consistent

Reply:
added kubectl, but I don't get what you mean in the second comment. Having command template and then example is in my view beneficial to understanding what needs to be changed.

Reply:
> Having command template and then example is in my view beneficial to understanding what needs to be changed.

my bad, didn't notice that it's only for the ones with an example output